src/schema/text_options.rs

```diff
@@ -25,8 +25,12 @@ pub struct TextOptions {
 
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 #[serde(untagged)]
+/// Enum to control how the fast field setting of a text field.
 enum FastFieldOptions {
+    /// Flag to enable/disable
     IsEnabled(bool),
+    /// Enable with tokenizer. The tokenizer must be available on the fast field tokenizer manager.
+    /// `Index::tokenizer_fast_field`.
     EnabledWithTokenizer { with_tokenizer: TokenizerName },
```

Comment:
Do we call it normalizer or something like that?

Reply:
I don't think we should restrict it to normalizer on tantivy side

Reply:
you are right